wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
 # Discord servers
 
-Index page for all articles regarding community [Discord](https://discord.com/) servers.
+As the community grows and expands, several [Discord](https://discord.com/) servers have been made and maintained by the community to specialise in the game's many different facets.
+
+Each server provides a space with a specific purpose in mind, where players get together to discuss and work on various different projects. This index aims to serve as an index where users can find a subcommunity they can best fit in.
```

Comment:
```suggestion
This article lists various [Discord](https://discord.com/) servers that have been created and maintained by the osu! community to host discussion spaces about the game's many different facets. Some are focused on the management of specific projects, while others are more general, but most of them include casual chat channels regardless.
```
maybe? the main thing I'm trying to do is make the first sentence introduce the article itself. rest is just rewording or what I would include here. I also removed the "aim" part at the end cuz i dont think this really needs to say a reason for why it's here and that may not necessarily be how the reader is using it

Reply:
Straighter to the point.